build.sbt

```diff
@@ -116,6 +118,7 @@ lazy val camel = akkaModule("akka-camel")
   .settings(Dependencies.camel)
   .settings(AutomaticModuleName.settings("akka.camel"))
   .settings(OSGi.camel)
+  .settings(AkkaDisciplinePlugin.coverageExclude)
 
 lazy val cluster = akkaModule("akka-cluster")
```

Comment:
is multi-jvm tests included in the reports?

Reply:
It has been thus far, though this afternoon I am tweaking the Jenkins config to hone in on mysterious failures.